src/SleepRepository.js

```diff
@@ -0,0 +1,51 @@
+import Sleep from './Sleep';
+
+class SleepRepository {
+  constructor(sleepData) {
+    this.allSleeps = sleepData;
+    this.individualsSleep = [];
+  }
+
+  getIndividualsSleep(id) {
+    this.individualsSleep = this.allSleeps.reduce((accum, sleep) => {
```

Comment:
Let's take a look at possibly making the `accum` the data type we are making, so in this instance it would be an array.